source/features/action-pr-link.tsx

```diff
@@ -0,0 +1,34 @@
+import * as pageDetect from 'github-url-detection';
+import select from 'select-dom';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+async function add(row: HTMLDivElement): Promise<void> {
+	const prNumber = select('a[data-hovercard-type="pull_request"]', row)?.textContent?.slice(1);
+
+	if (prNumber === undefined) {
+		return;
+	}
+
+	select('a.Link--primary', row)!.href += `?pr=${prNumber}`;
```

Comment:
I think needs something like this to be safe, just in case these links happen to have additional parameters in the future:

```suggestion
	const runLink = select('a.Link--primary', row)!;
	const parameters = new URLSearchParams(runLink.search);
	parameters.set('pr', prNumber);
	runLink.search = String(parameters);
```

I wish HTMLAnchorElement had a `searchParams` argument to make this easier 😒